apps/explorer/src/components/pagination/__test__/Pagination.test.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { render } from '@testing-library/react';
+import { describe, it, expect } from 'vitest';
+
+import Pagination from '../Pagination';
+
+describe('Pagination unit test', () => {
+    it('check pagination buttons', () => {
+        const { getByRole, getByTestId } = render(
+            <Pagination totalItems={105} itemsPerPage={10} currentPage={1} />
+        );
+        const nextBtn = getByRole('button', { name: /→/i });
```

Comment:
I don't love using these unicode properties for the lookup, can we use the test ID instead?

Reply:
We can not use the TestID in this instance because we have multiple elements with the same TestId present as the same time (mobile and desktop). I will update using querySelector